app/views/verify/address/index.html.slim

```diff
@@ -1,5 +1,14 @@
-p Pick an address verification mechanism
+h1.h2
+  = t('idv.titles.select_verification')
+p
+  = t('idv.messages.select_verification', sp_name: @sp_name)
```

Comment:
I don't think it would, since the user can definitely use login.gov without having their identity verified, right? Does it make sense to have this route only accessible during LOA3 verification? Or do we want to be able to verify a user's identity directly using login.gov in the future.

In that case, I think the copy will probably need to change somewhat.